packages/core/src/browser/source-tree/source-tree-widget.tsx

```diff
@@ -17,15 +17,24 @@
 import * as React from 'react';
 import { injectable, postConstruct, interfaces, Container } from 'inversify';
 import { DisposableCollection } from '../../common/disposable';
-import { TreeWidget, TreeNode, createTreeContainer, TreeProps, TreeImpl, Tree, TreeModel } from '../tree';
+import {
+    TreeWidget, TreeNode, createTreeContainer, TreeProps, TreeImpl, Tree, TreeModel
+} from '../tree';
 import { TreeSource, TreeElement } from './tree-source';
 import { SourceTree, TreeElementNode, TreeSourceNode } from './source-tree';
 
+const TREE_NODE_INDENT_WIDTH_SOURCE_CLASS = 'theia-tree-node-indent-width-source';
+
 @injectable()
 export class SourceTreeWidget extends TreeWidget {
 
     static createContainer(parent: interfaces.Container, props?: Partial<TreeProps>): Container {
-        const child = createTreeContainer(parent, props);
+        const sourceTreeProps = {
+            ...props,
+            nodeIndentWidthClassname: TREE_NODE_INDENT_WIDTH_SOURCE_CLASS
```

Comment:
it should be other way around, first provide default then spread that default can be overriden